api/lib/opentelemetry/distributed_context/propagation/http_text_format.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module DistributedContext
+    module Propagation
+      # HTTPTextFormat is a formatter that injects and extracts a value as text into carriers that travel in-band across
+      # process boundaries.
+      #
+      # Encoding is expected to conform to the HTTP Header Field semantics. Values are often encoded as RPC/HTTP request
+      # headers.
+      #
+      # The carrier of propagated data on both the client (injector) and server (extractor) side is usually an http request.
+      # Propagation is usually implemented via library-specific request interceptors, where the client-side injects values
+      # and the server-side extracts them.
+      class HTTPTextFormat
+        # TODO
```

Comment:
Is the implementation for `HTTPTextFormat` going to be part of this PR?

Reply:
That was my intent, although it's tempting to punt that to another PR so we can 🚢 this and regain some momentum.

Reply:
I'd be fine with that. I'm adding my approval. We can talk about this at the meeting and if nobody has any objections, we should merge it.